pkgs/by-name/wa/walker/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGo122Module,
+  fetchFromGitHub,
+  gtk4,
+  gtk4-layer-shell,
+  gobject-introspection,
+  pkg-config,
+}:
+
+buildGo122Module rec {
+  pname = "walker";
+  version = "0.0.18";
```

```suggestion
  version = "0.0.68";
```

Latest is 0.0.68, don't know if this is a mistake